### ISSUE TITLE
[A11yInspect] Fail: 2.4.3 A in /home/runner/work/test-app/test-app/src/sample.jsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#15]
## ❌ Accessibility Fail

**Rule:** 2.4.3 A
**File:** `/home/runner/work/test-app/test-app/src/sample.jsx`
**Line:** 1, **Column:** 1

### Description
Incorrect nesting of interactive HTML elements (Major)

### Context
```html
<form>
          dynamic-content
          <input type="text" placeholder="Name">

          dynamic-content
          <label>Email</label>
          <input type="email">

          dynamic-content
  
```

### Selector
`html > body > div > div > video > audio > form`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
